### ISSUE TITLE
Mind if I include your repo?

### ISSUE THREAD

**Issue by ifohancroft** [lpgera/pi-case-40-button#1]
Hi!

I love that you did an app so the button can be customized using cli!

I've created https://github.com/Pi-Case-40-Community/PiCase40 to hold community modifications for the Pi Case 40. Currently there are two GPIO replacement PCBs by me and another user, in different stages of completeness. Would you mind if I include your repo as a submodule there? It would be under Software/lpgera/pi-case-40-button

Thanks!

**Comment by lpgera** [comment]
Hey! That sounds like an awesome initiative for creating a collection of tools related to the case. Feel free to go ahead and include my repo! 🙂

**Comment by ifohancroft** [comment]
Thank you!

Oh, btw - would you mind renaming your branch from master to main? Let me know if/when you do, so I can change the submodule to track the new branch name.

**Comment by lpgera** [comment]
I fully support the goal of being inclusive, but I haven't had the time and capacity to rename all the master branches in my existing repositories, and _currently_ I'm not planning to do so.

Anyways, your question made me wonder why this relatively new repo of mine still uses the master branch as default, and I realized that it was created locally and the default name comes from my workstation's git configuration. I've updated it to use main going forward. 👍
